### PR TITLE
Toggle input alignment and Handle isEnable property for action type

### PR DESCRIPTION
**Pull request by IbrahimSulai**
This PR consist of the below changes:
1. Updated the alignment of toggle input as same as native platform.
2. Handled isEnabled property for action type, select action, and inline action - [https://github.com/microsoft/AdaptiveCards/issues/4670](https://github.com/microsoft/AdaptiveCards/issues/4670)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5760)

### CODE REVIEW

File: source/community/reactnative/src/components/inputs/input.js [hunk 1]
```diff
@@ -206,9 +206,10 @@ export class Input extends React.Component {
 			return null;
 		}
 		else {
+			let opacityStyle = { opacity: inlineAction.isEnabled == undefined ? 1.0 : inlineAction.isEnabled ? 1.0 : 0.4 };
```
Comment (vivekvijayakrishnan) [inline source/community/reactnative/src/components/inputs/input.js:209]: `undefined` check is not required as it's handled in **` BaseActionModel`**. can we remove this check?

File: source/community/reactnative/src/components/inputs/input.js [hunk 2]
```diff
@@ -236,11 +237,14 @@ export class Input extends React.Component {
 								value={this.props.value}
 							/>
 						</View>
-						<TouchableOpacity onPress={() => { this.onClickHandle(onExecuteAction, Constants.InlineAction) }}>
+						<TouchableOpacity
+							disabled={inlineAction.isEnabled == undefined ? false : !inlineAction.isEnabled} //isEnabled defaults to true
+							opacity={inlineAction.isEnabled == undefined ? 1.0 : inlineAction.isEnabled ? 1.0 : 0.5}
```
Comment (vivekvijayakrishnan) [inline source/community/reactnative/src/components/inputs/input.js:242]: We can remove the `undefined` check here as well.

**Review by vivekvijayakrishnan — CHANGES_REQUESTED** [review]
Please check the comments

File: source/community/reactnative/src/components/inputs/input.js (hunk 1 above)
Comment (IbrahimSulai) [reply source/community/reactnative/src/components/inputs/input.js:209]: inlineAction inside the payload is not inherited from the BaseActionModel. Only ActionModel is inherited from the BaseActionModel. So we need to handle the null cases in the places where we consumed.

File: source/community/reactnative/src/components/inputs/input.js (hunk 2 above)
Comment (IbrahimSulai) [reply source/community/reactnative/src/components/inputs/input.js:242]: inlineAction inside the payload is not inherited from the BaseActionModel. Only ActionModel is inherited from the BaseActionModel. So we need to handle the null cases in the places where we consumed.

**Review by vivekvijayakrishnan — APPROVED** [review]
LGTM

**Pull request merged**